tests/distributed/test_pipeline_parallel.py

```diff
@@ -181,6 +179,8 @@ def iter_params(self, model_name: str):
     "stabilityai/stablelm-3b-4e1t": PPTestSettings.fast(),
     "bigcode/starcoder2-3b": PPTestSettings.fast(),
     "upstage/solar-pro-preview-instruct": PPTestSettings.fast(tp_base=2),
+    "ai21labs/Jamba-tiny-dev": PPTestSettings.fast(),
+    "state-spaces/mamba-130m-hf": PPTestSettings.fast()
```

Comment:
Let's keep the alphabetical order of the models here (in line with model registry).

Reply:
Also, can you add one of these models to the list of models that are actually being tested in CI?

Reply:
Addressed